components/datadog/apps/tracegen/images/tracegen/main.go

```diff
@@ -99,6 +98,7 @@ func main() {
 		<-sigs
 		close(done)
 		fmt.Println("Exiting tracegen.")
+		time.Sleep(10 * time.Second) // alow time for other goroutines to shut down.
```

Comment:
How about adding a sync group or is it overkill? 😅

Reply:
Yeah, could do, but I *do* want some sort of timeout so it doesn't indefinitely run.
Not sure the sync will help here much, since the program will shut down as soon as the main function exits, so this sleep won't actually delay the shutdown, just act as a safety to kill the process if anything deadlocks.

LMK.